gnomad/sample_qc/relatedness.py

```diff
@@ -720,7 +720,8 @@ def filter_mt_to_trios(mt: hl.MatrixTable, fam_ht: hl.Table) -> hl.MatrixTable:
 
     mt = mt.filter_cols(hl.is_defined(fam_ht[mt.col_key]))
     mt = filter_to_autosomes(mt)
-    mt = annotate_adj(mt)
+    if "adj" not in list(mt.entry):
```

Comment:
I think you might not need `in list(mt.entry)`; `in mt.entry` should work (but not 100%)